cnd/src/http_api.rs

```diff
@@ -189,7 +180,7 @@ fn make_order_entity(properties: OrderProperties) -> Result<siren::Entity> {
 }
 
 fn cancel_action(order: &OrderProperties) -> Option<siren::Action> {
-    if order.state.open > 0 {
+    if order.state.open != Amount::btc(btc_zero().to_inner()) {
```

Comment:
```suggestion
    if order.state.open != Amount::btc::ZERO {
```
If it does not exist yet, create this const :)

Reply:
Also, a helper method `State::is_open` may make more sense here instead of exposing the internals. If such helper was existing this line would not have changed ;)